### DISCUSSION TITLE
Faire un ghost ping pour les commandes qui demandent une interaction avec une autre personne

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#1270] [Suggestions fermées]
En gros, pour les commandes qui vont demander une interaction, comme fight, pettrade, autre (proposez), on devrait faire un ghost ping (send un ping qu'on delete direct ensuite) afin que le ou les joueurs soient notifiés qu'une action les concernant est attendue

**Comment by BastLast** [comment]
Horrible super mal géré par les clients discord

**Comment by ghost** [comment]
des gens avec des mauvaises intentions risquent de ghost ping uniquement dans le but de nuire aux joueurs

**Comment by romain22222** [comment]
alors attendez je vais reformuler :

il faudrait simuler un ping aux endroits nécessaires où le ping était un moyen d'avertissement que l'action te concerne

**Comment by ghost** [comment]
suffit de faire une commande et d'annuler derrière et faire ca en boucles
il me semble que les ghosts pings sont contre les ToS de discord

**Comment by BastLast** [comment]
nan mais ajouter une mention au dessus de l'embed pr ping why not mais pas un ghost ping

**Comment by BastLast** [comment]
je refuse l'idée, si vous voulez en créer une mieux formulée amusez vous :)
